analysis/map.R

```diff
@@ -8,29 +8,58 @@ exposures_by_hasc <- risks_table_mapped %>%
   group_by(hasc) %>%
   summarize(exposures = sum(exposure))
 
+bra_layer0 <- sf::st_read("external_data/gadm36_BRA_gpkg/gadm36_BRA.gpkg", layer = "gadm36_BRA_0")
 bra_layer1 <- sf::st_read("external_data/gadm36_BRA_gpkg/gadm36_BRA.gpkg", layer = "gadm36_BRA_1")
 
 brazil <- bra_layer1 %>%
   left_join(exposures_by_hasc, by = c(HASC_1 = "hasc"))
 
 bins <- c(0, 10^3,10^4, 10^5, 10^6, Inf)
 pal <- colorBin("YlOrRd", domain = brazil$exposures, bins = bins)
+<<<<<<< HEAD
```

Comment:
These `<<<<`, `====`, etc. should be deleted after you've selected what you want to keep. They're there to delineate diverged code from different branches.

Reply:
Woops, should be fixed now.  Thought I had deleted them.